src/routes/wallet/deposit.tsx

```diff
@@ -39,14 +58,77 @@ export default class Deposit extends React.Component<DepositProps, any> {
   @observable
   value: number | string = 0;
 
+  @observable
+  public pendingDeposits: { [key:string]:PendingDeposit } = {};
+
+  private storageKey: string;
+
+  constructor(props) {
+    super(props);
+
+    this.storageKey = `pendingDeposits-${props.exitHandler.address}`;
+    props.exitHandler.events.on('NewDeposit', this.handleNewDeposit);
+
+    reaction(
+      () => props.unspents.list,
+      () => this.checkPendingDeposits()
+    );
+
+    reaction(() => this.selectedToken, () => this.loadPendingDeposits);
+
+    this.checkPendingDeposits();
+  }
+
+  @autobind
+  private loadPendingDeposits() {
+    const { color } = this.props;
+    this.pendingDeposits = storage.load(this.storageKey).filter(dep => dep.color === color);
+  }
+
+  private storePendingDeposits() {
```

Comment:
You can use reactions to call this method

Reply:
Or even better — use `autorun`

Reply:
Reaction is not working for me 🤔

```
reaction(() => this.pendingDeposits, this.storePendingDeposits);
```

never reacts. Tried to add comparator as well

Reply:
autorun works though 🎉